pkg/cache/cache.go

```diff
@@ -59,6 +60,11 @@ type Deleter interface {
 	Delete(key string) error
 }
 
+type Counter interface {
+	Increment(key string) (int64, error)
+	PFAdd(key string, els []string) (int64, error)
```

Comment:
We should change the interface to:

E.g. `PFAdd(key string, els ...string) (int64, error)`

Also in here:
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/redis/v3/redis.go#L66

Reference:
https://pkg.go.dev/github.com/fiorix/go-redis/redis#Client.PFAdd